.github/workflows/check-coverage-deploy.yaml

```diff
@@ -56,27 +123,40 @@ jobs:
 
       - run: ls
       - run: ls ${PKG_NAME}
-      - run: ls drat
       - run: echo ${{ github.ref }}
-      - name: Check to deploy
-        if: ${{ endswith(github.ref, '/master') || endswith(github.ref, '/main')  }}
+
+      - name: Set environmental variables 
         run: |
           cd ${PKG_NAME}
           Rscript \
```

Comment:
Let's use the github action syntax of `shell: Rscript {0}`, instead of just `run: Rscript -e "..."`

Reply:
Switched and testing!

Reply:
Have you checked the entire yaml file?

Reply:
Gotcha, I'll do it in the dependencies install as well!